aleph-client/src/multisig.rs

```diff
@@ -91,7 +94,7 @@ struct Multisig {
 
 /// This represents the ongoing procedure of aggregating approvals among members
 /// of multisignature party.
-#[derive(Debug)]
+#[derive(Clone, Eq, PartialEq, Debug)]
```

Comment:
[maybe stupid] I remember that I intentionally didn't make it `Clone` - such object was supposed to be created from `initiate_aggregation_with_hash`/`initiate_aggregation_with_call` and then, passed and destroyed with consecutive calls to `approve` (see the method signature - it takes `sig_agg: SignatureAggregation`); the intention was to never reuse `SignatureAggregation`, because it would lead to an unnecessary extrinsic submission

however, I won't argue that this care is not an overkill :)

Reply:
I would say that if this is semantically not supposed to be copied, then we definitely should not derive `Clone` for it.